Failing CI on Windows for sari and wiki_split metrics

## Describe the bug
Our CI is failing from yesterday on Windows for metrics: sari and wiki_split
```
FAILED tests/test_metric_common.py::LocalMetricTest::test_load_metric_sari - ...
FAILED tests/test_metric_common.py::LocalMetricTest::test_load_metric_wiki_split
```

See: https://app.circleci.com/pipelines/github/huggingface/datasets/11928/workflows/79daa5e7-65c9-4e85-829b-00d2bfbd076a/jobs/71594

--- a/setup.py
+++ b/setup.py
@@ -150,6 +150,7 @@ TESTS_REQUIRE = [
     "mauve-text",
     "rouge_score",
     "sacrebleu",
+    "sacremoses",
     "scikit-learn",
     "scipy",
     "sentencepiece",  # for bleurt

--- a/setup.py
+++ b/setup.py
@@ -146,14 +146,14 @@ TESTS_REQUIRE = [
     "zstandard",
     # metrics dependencies
     "bert_score>=0.3.6",
+    "jiwer",
+    "mauve-text",
     "rouge_score",
     "sacrebleu",
-    "scipy",
-    "seqeval",
     "scikit-learn",
-    "jiwer",
+    "scipy",
     "sentencepiece",  # for bleurt
-    "mauve-text",
+    "seqeval",
     # to speed up pip backtracking
     "toml>=0.10.1",
     "requests_file>=1.5.1",

